### PR TITLE
feat:(feedback-spa) - Feedback SPA readiness on vuejs stack

### PR DESCRIPTION
**Pull request by riginoommen**
# Resolves 836, 864, 862, 864, 866

# Explain the feature
Created the Feedback SPA with vuejs which supports listing of feedback from multiple sources in single view.

* Exporting feedback.
* Filtered view of feedback.
* Consolidated view of feedback

## Does this PR introduce a breaking change

Yes
## Screenshots

<details>
<summary>View Screenshots</summary>

feedbackSPA

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by riginoommen** [comment]
Please review this i am making this as a WIP PR because i have a deployment plan for the current feedback. So i request to please consider it for review

**Review by hybridx — APPROVED** [review]
LGTM

File: packages/feedback-spa/README.md [hunk 1]
```diff
@@ -1,23 +1,57 @@
-# Feedback
+# Feedback SPA
+Feedback SPA provides the consolidated view of user experience which has shared to developer team.
```
Comment (deshmukhmayur) [inline packages/feedback-spa/README.md:2]: Can you update readme with a short description?

File: packages/feedback-spa/spaship.yaml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-# SPAship config file
```
Comment (deshmukhmayur) [inline packages/feedback-spa/spaship.yaml:1]: Add the spaship.yaml file back. This needs to be in source control to ensure consistent deployments.

**Review by deshmukhmayur — CHANGES_REQUESTED** [review]
Can you also remove the vue.js's default favicon from the page?

**Review by ghanlohar — APPROVED** [review]
Looks good.

File: packages/feedback-spa/README.md (hunk 1 above)
Comment (riginoommen) [reply packages/feedback-spa/README.md]: updated

File: packages/feedback-spa/spaship.yaml (hunk 2 above)
Comment (riginoommen) [reply packages/feedback-spa/spaship.yaml]: Included it back

File: packages/feedback-spa/README.md (hunk 1 above)
Comment (deshmukhmayur) [reply packages/feedback-spa/README.md]: Also add a newline after the heading.

Comment (riginoommen) [reply packages/feedback-spa/README.md]: Added

**Review by deshmukhmayur — APPROVED** [review]
Good to merge

**Pull request merged**